internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr_test.go

```diff
@@ -152,7 +149,16 @@ func TestKafkaManager_Reconcile(t *testing.T) {
 					},
 					ListAllFunc: func() (dbapi.KafkaList, *errors.ServiceError) {
 						return dbapi.KafkaList{
-							testutils.NewKafkaRequest(testutils.WithDefaultTestValues()),
+							{
+								Meta: api.Meta{
+									ID: "test",
+								},
+								Name:         "test",
+								ClusterID:    "test",
+								Status:       constants.KafkaRequestStatusAccepted.String(),
+								InstanceType: "unsupported-instance-type",
+								SizeId:       "unsupported-size-id",
+							},
```

Comment:
I believe we can keep the `NewKafkaRequest` utility. It can be useful for other tests as well.

Reply:
I've added back the change in a separate commit.  
Can you have a look? @ziccardi thanks

side note: ideally, in the initial pull request, this change should have been a separate commit, then reverting would have been just reverting the temporary commit.

Reply:
@ziccardi I've also brought back the usage of NewKafkaRequest utility in this test. I think the PR is ready to be merged!